assets/composer/settings.lando.php

```diff
@@ -32,8 +32,6 @@
  *
  * @see https://wiki.php.net/rfc/expectations
  */
-assert_options(ASSERT_ACTIVE, TRUE);
```

Comment:
Out of interest has this been replaced by setting `zend.assertions` to 1 somewhere?